### ISSUE TITLE
Alerts: Add linter to CI

### ISSUE THREAD

**Issue by ArthurSens** [gitpod-io/gitpod#13246]
When we used Jsonnet to create alerts, we used a small jsonnet script and [promtool](https://github.com/prometheus/prometheus/tree/main/cmd/promtool) to make sure all alerts are working. We lost this validation check once we migrated to YAML. As a consequence, we've been merging broken alerts and only noticing it by looking at prometheus-operator logs.

We should re-implement this validation for all the YAML files

**Comment by ArthurSens** [comment]
Hey @meysholdt, I've noticed that this was moved from "Continuous Delivery" to "monitoring and observability". Well, looking at the team's structure now, I don't see how this falls under "Operational Experience" either 😅. It would be nice if we could sync on issues before just throwing it around inboxes 😛 

That said, could we pair on this one at some point? I believe this linter will have to work with leeway, with which I have close to 0 experience, and also with promtool, which I do have experience with. Maybe if we work together we can get this one done quicker :)
